### PR TITLE
Docs Migration to GitHub

### PR DESCRIPTION
**Pull request by mdroidian**
The [Wiki](https://github.com/dvargas92495/roamjs-smartblocks/wiki) is up as well to explore and get a feel for.

Initially I thought it could have been an option, but I'm leaning back towards `/docs` now based on your comments from today about having the docs updates in the same commit as the code as well some of the limitations of the wiki.

Negatives of Wiki
- docs not versioned along with code
- docs not available locally when someone clones the repo
- docs don't get reviewed
- can't upload media, would have to store it in repo anyway

The negatives of `/docs` would be minor
 - creating and maintaining an index ToC on all pages (if we want, I lean yes)
 - creating and maintaining a footer on all pages (if we want, I lean no)
 - No "quick access" to docs from `Wiki` tab at top
   - which we could mitigate by adding a single page directing them to `/docs`

We probably could create GH Actions to sync to the Wiki, but `/docs` only seems fine.

A couple things to look at in this draft:
 - naming convention of files to keep order and allow for later insertions
   - acts as supplementary navigation
   - image

 - `/docs/media` for images, gifs, videos

ToDo
- [x] move all images
- [x] update internal links
- [x] create ToC / footer
- [x] delete wiki pages

### CODE REVIEW

**Comment by dvargas92495** [comment]
> creating and maintaining an index ToC on all pages (if we want, I lean yes)
> creating and maintaining a footer on all pages (if we want, I lean no)

Both of these could also be mitigated via scripting

**Comment by dvargas92495** [comment]
> naming convention of files to keep order and allow for later insertions

This is pretty dope - would actually be a fan of doing this for samepage docs instead of the `metadata.json` files that currently exist there

File: .gitignore [hunk 1]
```diff
@@ -10,6 +10,5 @@ store
 store.js
 main.js
 extension.js
-*.png
```
Comment (mdroidian) [inline .gitignore:13]: fyi, this was removed.
It didn't seem to affect any other files outside of docs/media

File: docs/migration-guide.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+  location: { "parent-uid": parent_uid, order: order + bBelow },
+  block: { string: block_string, uid },
+});
+return uid;
+```
+
+- `roam42.common.moveCursorToNextBlock`
+  - This method is pretty hacky and would not recommend using until Roam releases an API for doing this. In any case, if you're still interested, I would look at [the source code directly](https://github.com/dvargas92495/roam42/blob/master/common/commonFunctions.js#L317-L330).
+- `roam42.common.moveCursorToPreviousBlock`
+  - This method is pretty hacky and would not recommend using until Roam releases an API for doing this. In any case, if you're still interested, I would look at [the source code directly](https://github.com/dvargas92495/roam42/blob/master/common/commonFunctions.js#L332-L344).
```
Comment (mdroidian) [inline docs/migration-guide.md:145]: Where are these files located?

**Comment by mdroidian** [comment]
Didn't `npm version patch`, will do with follow up PR

File: docs/migration-guide.md (hunk 2 above)
Comment (dvargas92495) [reply docs/migration-guide.md:145]: You'll have to go back into the history for workbench, and use a commit snapshot (which is what I should've done)

**Comment by dvargas92495** [comment]
So dope - thanks for taking this on!

**Comment by dvargas92495** [comment]
so dope - thanks for taking this on!

**Pull request merged**